.travis.yml

```diff
@@ -15,6 +15,8 @@ install:
   - export PATH="$HOME/gopath/bin:$PATH"
   - echo http://localhost > $HOME/.tsuru_target
   - make get GO_EXTRAFLAGS=-x
+before_script:
+  - nc -zvv localhost 27017; while [[ $! -ne 0 ]]; do nc -zvv localhost 27017; done
```

Comment:
Maybe add `echo` and `sleep` so that we can see when it retries and how long it takes?